charts/aspnetcore/templates/deployment-hpa.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if (((.Values.deployment).autoscaling).enabled) }}
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ .Release.Name }}-hpa
```

Comment:
We usually allow the name to be set in values if people want

Reply:
is that feature used ?

Reply:
I've seen it somewhere from what I remember